packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.tsx

```diff
@@ -40,24 +41,24 @@ export const TEXT_ALIGN = {
   start: 'start',
   center: 'center',
   end: 'end',
-};
+} as const;
 
 export type Props = {
```

Comment:
I think it was already discussed, so just confirming, the approach we take is exporting component props, instead of inferring them:
```tsx
import type { Props as BpkGraphicPromoProps } from 'bpk-component-graphic-promotion';
// vs
import { BpkGraphicPromo } from 'bpk-component-graphic-promotion';
type BpkGraphicPromoProps = React.ComponentProps<typeof BpkGraphicPromo>
```

Reply:
This is a very good point.

This was kept as it is purely for similarity to the existing component API, with no deeper thinking.

The code snippet definitely works (just tested with the beta package to make sure in that assertion).

The downside is that every consumer needs to set up the type themselves, rather than it being provided. 

I can go either way on this - any preference @olliecurtis? (/Ana)

cc: @kirstybryce